src/cozmo/conn.py

```diff
@@ -332,7 +350,8 @@ def _handle_ui_device_connected(self, msg):
                     # App is newer
                     logger.error(
                         'Please update your SDK to the newest version by calling command: '
-                        '"pip3 install --user --ignore-installed cozmo"')
+                        '"pip3 install --user --ignore-installed cozmo" '
```

Comment:
Like above, why are we using --ignore-installed, since that does a reinstall? And should say "download" instead of "downloading".